<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeformationLocalWindow</class>
 <widget class="QDialog" name="DeformationLocalWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" rowstretch="2,2,1,1,2,1,1" columnstretch="2,5">
     <item row="5" column="0">
      <widget class="QLabel" name="labelDegree">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Степень:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Вид:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="labelDeformationLocal">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Локализация деформации:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="buttonOk">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBoxSide">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Нет</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Левая</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Правая</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>И левая и правая</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="labelPreviouslyInstalled">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <underline>false</underline>
        </font>
       </property>
       <property name="text">
        <string>Если установлена ранее:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBoxView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Не определено</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Сколиоз</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Кифоз</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Лордоз</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ротация</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelSide">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Сторона:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="comboBoxDegree"/>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QComboBox" name="comboBoxDeformationLocalInfo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Кранивертебральные</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Шейные</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Шейно-грудные</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Верхнегрудные</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Среднегрудные</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Нижнегрудные</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Груднопоясничные</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Поясничные</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Пояснично-кресцовые</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
